op-acceptor/ui/tree.go

```diff
@@ -0,0 +1,118 @@
+package ui
+
+import (
+	"strings"
+	"unicode/utf8"
+)
+
+// Tree hierarchy symbols using box drawing characters
+const (
+	// Basic tree connectors
+	TreeBranch     = "├── " // Branch connector (tee right + horizontal line + space)
+	TreeLastBranch = "└── " // Last branch connector (bottom left corner + horizontal line + space)
+	TreeVertical   = "│"    // Vertical line for continuing hierarchy
+
+	// Spacing patterns for different indentation levels
+	TreeContinue   = "│   "     // Vertical line + 3 spaces (parent has more siblings)
+	TreeIndent     = "    "     // 4 spaces (parent was last, no vertical line needed)
+	TreeDeepIndent = "│       " // Vertical line + 7 spaces (deeper nesting)
+	TreeMultiLevel = "│   │   " // Multiple vertical lines for complex hierarchy
+
+	// Box drawing characters for borders/containers
+	BoxTopLeft     = "┌"
+	BoxTopRight    = "┐"
+	BoxBottomLeft  = "└"
+	BoxBottomRight = "┘"
+	BoxVertical    = "│"
+	BoxHorizontal  = "─"
+	BoxTeeRight    = "├"
+	BoxTeeLeft     = "┤"
+
+	// Suite-specific patterns (with additional indentation)
```

Comment:
nit: in theory these shouldn't be needed, as they're composite symbols (indent + tree connector)
Removing them would avoid having use-case constructs (suites) creep into this package.
Anyways, no big deal :)

Reply:
Yeah. Will address in a follow-up!
Thanks! <3